src/screens/updatePlan/CreatePlanItemButton.tsx

```diff
@@ -59,3 +18,46 @@ const styles = StyleSheet.create({
     flex: 1,
   },
 });
+
+export const CreatePlanItemButton: SFC<Props> = (props: Props) => {
```

Comment:
What is rationale behind this change?
- Functional components does not support hot reloading
- Pure components are better for performance, as they do only shallow prop comparison (might not apply to this particular component)

What do we gain in return?

Reply:
Since I have mostly standard React background, I am not familiar with hot reloading. What about that https://github.com/bvic23/babel-plugin-functional-hmr? Maybe it can fix the problem with functional components?

What do we gain in return?
- We can use `useCallback` hook for memoizing the callback method, and avoid the anonymous function declaration in every component re-render. This is a performance bottleneck.
- We get rid of useless `this` keyword for this component. 
- In this particular scenario shallow comparison does not give us any benefits.

Reply:
It seems that hooks, sfc, and hot reloading don't like each other. I will dig into that deeper tomorrow, since my knowledge on that topic is negligible.